UnitTest-ColorPickerUI/Helpers/ColorHelperTest.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Drawing;
+using ColorPicker.Helpers;
+using Xunit;
```

Comment:
Can we shift to MS Test so we are heavily fragmenting our unit test frameworks?  DataRow would be the equivalent item here

Reply:
Address feedback -> switch to MsTest framework